app/views/user/index.html.erb

```diff
@@ -1 +1,5 @@
-<h1>index user</h1>
```

Comment:
Please follow this [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_root_users.png) to show all users on `app/views/users/index.html.erb`. 
All users page of your project looks something like this:
![show](https://user-images.githubusercontent.com/35504149/151187590-5b8d72b9-88b8-4225-b4a3-27eb9242ac31.png)